RadixWallet/Features/HomeFeature/Coordinator/Home+View.swift

```diff
@@ -113,7 +113,7 @@ private extension View {
 			state: /Home.Destination.State.exportMnemonic,
 			action: Home.Destination.Action.exportMnemonic,
 			content: { childStore in
-				NavigationView {
+				NavigationStack {
```

Comment:
Does this have any effect on toolbar visibility? :)

Reply:
Speaking of toolbar, is the idea to show the completion toolbar here, or is that too much work?

Reply:
The toolbar is not needed though in this case, I don't show the word completion when verifying the mnemonic.